src/Features/Core/Portable/QualifyMemberAccess/AbstractQualifyMemberAccessDiagnosticAnalyzer.cs

```diff
@@ -147,14 +147,16 @@ private void AnalyzeOperation(OperationAnalysisContext context, IOperation opera
             }
         }
 
-        private bool IsStaticMemberOrTargetMethod(IOperation operation)
+        private bool IsMemberOrTargetMethodStaticOrLocalFunction(IOperation operation)
         {
             switch (operation)
             {
                 case IMemberReferenceOperation memberReferenceOperation:
-                    return memberReferenceOperation.Member == null || memberReferenceOperation.Member.IsStatic;
+                    return memberReferenceOperation.Member == null || memberReferenceOperation.Member.IsStatic ||
+                        memberReferenceOperation.Member is IMethodSymbol memberMethod && memberMethod.MethodKind == MethodKind.LocalFunction;
```

Comment:
consider using a pattern like `memberReferenceOperation.Member is IMethodSymbol { MethodKind: MethodKind.LocalFunction }`

Reply:
Thank you for the tip! Updated.